queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
- [ ] Kindly be noticed that the question is `find all animals whose name ends in "mon"?` so you should write the query like this.
```sql
`SELECT * FROM animals WHERE name LIKE '%mon';`
```

Reply:
@ShahierNashaat , Thank you for the correction.